cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,36 +1,40 @@
-
 import { AddressStepPage, LoginPage, MenuContentPage, PaymentStepPage, ProductsListPage, ShippingstepPage, ShopingCartPage } from "../page/index"
 
-const menuContentPage = new MenuContentPage()
-const productsListPage = new ProductsListPage();
-const shopingCartPage = new ShopingCartPage()
-const loginPage = new LoginPage()
-const addressStepPage = new AddressStepPage()
-const shippingstepPage = new ShippingstepPage()
-const paymentStepPage = new PaymentStepPage()
-
 describe("Buy a t-shirt", () => {
 
-    it("then the t-shirt should be bought", () => {
+  const menuContentPage = new MenuContentPage()
+  const productsListPage = new ProductsListPage()
+  const shopingCartPage = new ShopingCartPage()
+  const loginPage = new LoginPage()
+  const addressStepPage = new AddressStepPage()
+  const shippingstepPage = new ShippingstepPage()
+  const paymentStepPage = new PaymentStepPage()
+
+  it("then the t-shirt should be bought", () => {
 
-      menuContentPage.visitMenuContentPage()
-      menuContentPage.goToTShirtMenu()
+    const tShirtsNamesArray: string[] = ["Faded Short Sleeve T-shirts"];
+    const sThirtsNumber = 1; 
 
-      productsListPage.selectTShirtOfProductsList()
+    menuContentPage.visitMenuContentPage()
+    menuContentPage.goToTShirtMenu()
+    
+    productsListPage.validateItemsNumber(sThirtsNumber)
+    productsListPage.validateItemsNames(tShirtsNamesArray)
+    productsListPage.selectTShirtOfProductsList()
 
-      shopingCartPage.addTShirtToCart()
-      shopingCartPage.proceedTShirtSelectInCart()
+    shopingCartPage.addTShirtToCart()
+    shopingCartPage.proceedTShirtSelectInCart()
 
-      loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor")
+    loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor")
 
-      addressStepPage.confirmAddress()
+    addressStepPage.confirmAddress()
 
-      shippingstepPage.acceptTerms()
-      shippingstepPage.proceedCarrier()
+    shippingstepPage.acceptTerms()
+    shippingstepPage.proceedCarrier()
 
-      paymentStepPage.paybyBankWire()
-      paymentStepPage.confirmPay()
-      paymentStepPage.getConfirmationTitle().should("have.text", "Your order on My Store is complete.")
+    paymentStepPage.paybyBankWire()
+    paymentStepPage.confirmPay()
+    paymentStepPage.getConfirmationTitle().should("have.text", "Your order on My Store is complete.")
```

Comment:
this is the Assert of the AAA, in this case, you can separate it with a blank line from the other methods